packages/react-ui/src/widgets/CategoryWidgetUI.js

```diff
@@ -428,9 +466,10 @@ function CategoryWidgetUI(props) {
       <Grid container item className={classes.categoriesWrapper}>
         {[...Array(4)].map((_, i) => (
           <Grid key={i} container direction='row' spacing={1} className={classes.element}>
-            <Grid container item xs>
+            <Grid container item xs zeroMinWidth>
               <Grid container item direction='row' justifyContent='space-between'>
-                <Typography variant='body2'>
+                <Typography variant='body2' noWrap>
+                  plop
```

Comment:
hey @atoom75, just wondering if this `plop` is here in purpose?

Reply:
Fixed, sorry 😅